### PR TITLE
Download Latin.unicharset along with radical-stroke.txt

### PR DESCRIPTION
**Pull request by Shreeshrii**
Need another PR to add Inherited.unicharset after https://github.com/tesseract-ocr/langdata_lstm/pull/41/ is merged

### CODE REVIEW

**Review by kba — APPROVED** [review]

**Comment by stweil** [comment]
All unicharset files for scripts are potentially needed, starting with `Arabic.unicharset` and ending with `Thai.unicharset`.

I usually get the required ones to satisfy the error message(s), but still don't know what happens if they are missing.

**Comment by Shreeshrii** [comment]
I added only Latin and Inherited unicharsets in this list because these are required in almost all cases, even though they don't stop processing like missing radical-stroke.txt.

We could add another optional variable for SCRIPT_UNICHARSET, downloading it when it is non-blank.

>still don't know what happens if they are missing.

I think some characters e.g. Arabic accents get dropped in the generated unicharset by unicharset_extractor.  That was the reason I built the Inherited.unicharset.

**Review by stweil — CHANGES_REQUESTED** [review]

**Comment by stweil** [comment]
A list of all required `unicharset` files can be extracted from `unicharset`:

    cat $(OUTPUT_DIR)/unicharset | sed s/.*0,0,0.// | sed 's/ .*//' | sort | uniq | grep "^[A-Z][a-z][a-z]*" | grep -v common

File: Makefile [hunk 1]
```diff
@@ -303,6 +303,8 @@ $(OUTPUT_DIR).traineddata: $(LAST_CHECKPOINT)
 endif
 
 $(DATA_DIR)/radical-stroke.txt:
+#	wget -O $(DATA_DIR)/Inherited.unicharset 'https://github.com/tesseract-ocr/langdata_lstm/raw/master/Inherited.unicharset'
+	wget -O $(DATA_DIR)/Latin.unicharset 'https://github.com/tesseract-ocr/langdata_lstm/raw/master/Latin.unicharset'
```
[the remark this answers is not in the document]
Comment (Shreeshrii) [reply Makefile:307]: Inherited.unicharset is NOT there in langdata_lstm repo. I created it by copying the lines with Inherited from other unicharsets. But there are some differences in coordinates for same character in different unicharsets, so I am not sure which one is to be used.

**Comment by Shreeshrii** [comment]
Thanks for the suggestions @stweil and the hint to get the list of required unicharsets from  $(OUTPUT_DIR)/unicharset. 

I am having a hard time putting it together in a separate Makefile target using the list. Would appreciate if you can make the required change.

**Comment by wrznr** [comment]
@kba Could you pls. have a look at the change request and maybe come up with a proposal?

**Comment by Shreeshrii** [comment]
I added ` sed '/Common/d' | sed '/Inherited/d' | sed '/Joined/d'` to the script suggested by @stweil because there are no unicharsets for Common and Inherited . Joined was being picked up accidentally.

A simpler way maybe asking the user to specify a script and download that.

**Comment by Shreeshrii** [comment]
>A simpler way maybe asking the user to specify a script and download that.

I have tried that in the new [Makefile-font2model](https://github.com/tesseract-ocr/tesstrain/blob/1466135f0ec673ec5cb25385d76241de24058511/Makefile-font2model#L205-L213)
I think that is a much cleaner way of doing this.

**Comment by Shreeshrii** [comment]
Included as part of https://github.com/tesseract-ocr/tesstrain/pull/230

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (typeoo) [reply Makefile]: Hi 
how can I get the **Inherited.unicharset**

**Pull request closed without merging**